### ISSUE TITLE
Flash writer not writing the flash dump file

### ISSUE THREAD

**Issue by cooltutel** [aldostools/flashwriter#1]
My version was 4.91.
I start the writing proccess
it tells me to check iff it wrote the flash dump file
i check it
and...
![image](https://github.com/user-attachments/assets/0e1128c0-a0ec-48ac-b44c-54832430c342)
Where the god did it go?
It was not even written.

**Comment by aldostools** [comment]
1. You need to have [HFW 4.91](https://mediafire.com/file/vvqd18e9emjdb44) installed. It doesn't work on OFW 4.91.
2. You need an USB flash drive formatted in FAT32 MBR connected to port `/dev_usb000` (the one closest to the Bluray drive)
3. The dump file should be written to `/dev_usb000/PS3FlashDump491OFW.bin`
4. Follow the tutorials in [this post](https://x.com/aldostools/status/1814693451327213996) for 3 alternative methods.

**Comment by cooltutel** [comment]
1. It is HFW.
2. It is formatted to FAT32 MBR, connected to /dev_usb000.
3. It should be tho...
4. Im diign t5he instructions from that video.

**Comment by cooltutel** [comment]
> 1. You need to have [HFW 4.91](https://mediafire.com/file/vvqd18e9emjdb44) installed. It doesn't work on OFW 4.91.
> 2. You need an USB flash drive formatted in FAT32 MBR connected to port `/dev_usb000` (the one closest to the Bluray drive)
> 3. The dump file should be written to `/dev_usb000/PS3FlashDump491OFW.bin`
> 4. Follow the tutorials in [this post](https://x.com/aldostools/status/1814693451327213996) for 3 alternative methods.

it is HFW.
It is formatted to FAT32 MBR, connected to /dev_usb000.
It should be tho...
Im diign t5he instructions from that video.

**Comment by aldostools** [comment]
VERY IMPORTANT: Do not use the Russian HFW 4.91 to avoid a brick.
The patch `flash491.P3T` was built based on Joonie's HFW 4.91
MD5 of PS3UPDAT.PUP: `C2717C377EB4F0FF458B8819B1E53D50`

Flash writer should work If you follow the video and your console is CFW compatible (all phat models and slims 20xx or 21xx). 
It works on 25xx models only if they have min FW 3.56 or lower. It doesn't work with superslims or slims 3xxx.

**Comment by cooltutel** [comment]
My serial number starts with CECHK. It is NOR.

**Comment by aldostools** [comment]
CECHK is a phat model. All phats are CFW compatible.

If you're having issues with the flash writer, use ps3toolset.com/bgtoolset
Follow this tutorial
https://www.psx-place.com/threads/bgtoolset-access-workaround-guide.44181/

Here is a video tutorial if you prefer
https://www.youtube.com/watch?v=LLyf9L18wKk

**Comment by cooltutel** [comment]
The site gives an error (PS3 Toolset)

**Comment by aldostools** [comment]
Yes there is a known issue with certificates used by ps3toolset.com

You must follow the tutorials linked in my previous reply to bypass the error using the Fiddler method suggested by NiQ.

If you want to use PS3Xploit Flashwriter 4.91, I recommend you to follow this video tutorial
[![](https://i.ytimg.com/vi/rwqk_8mJy30/hqdefault.jpg)](https://www.youtube.com/watch?v=rwqk_8mJy30)

**Comment by cooltutel** [comment]
i sadly used HEN instead of CFW.

**Comment by aldostools** [comment]
https://ps3toolset.com is back again.
